Sources/Utility/Process.swift

```diff
@@ -95,6 +105,9 @@ public final class Process: ObjectIdentifierProtocol {
 
     /// The output read from the process, if redirected.
     private var output: [Int8] = []
+    
+    /// Queue to protect concurrent reads.
+    private let serialQueue = DispatchQueue(label: "org.swift.swiftpm.process")
```

Comment:
From the docs it seems the label is only used for uniquely identifying during debugging, however I think it would be a better experience if we suffix it with something to uniquely identify the instance. Do you have any recommendations ?